### PR TITLE
Support latest VMService version.

### PR DESCRIPTION
**Pull request by jacob314**

### CODE REVIEW

**Comment by jacob314** [comment]
@bkonyi just pushed a new vm service version and we are broken without this change on pub upgrade.

**Comment by jacob314** [comment]
TBR

File: packages/devtools_app/lib/src/vm_service_wrapper.dart [hunk 1]
```diff
@@ -900,6 +878,24 @@ class VmServiceWrapper implements VmService {
       trackFuture('getRetainingPath',
           _vmService.getRetainingPath(isolateId, targetId, limit));
 
+  @override
+  Future<CpuSamples> getAllocationTraces(String isolateId,
```
Comment (terrylucas) [inline packages/devtools_app/lib/src/vm_service_wrapper.dart:882]: The old API calls to getAllocationSamples needs to be renamed to use getAllocationTraces and the old entry removed.  I'll create another PR to do this.

**Review by terrylucas — COMMENTED** [review]
LGTM, thanks.

File: packages/devtools_app/lib/src/vm_service_wrapper.dart (hunk 1 above)
Comment (kenzieschmoll) [reply packages/devtools_app/lib/src/vm_service_wrapper.dart:882]: commented on https://github.com/flutter/devtools/pull/2720/files#r579327782 - we probably need to call getAllocationSamples here for older versions of the protocol

**Pull request merged**